File: decoder.js
const crypto = require('crypto');
const fs = require('fs');

// Load the key and IV from the file
const keyFile = fs.readFileSync('key.keyzor');
const key = keyFile.slice(0, 16);
const iv = keyFile.slice(16, 32);

// Load the encrypted file
const encryptedFile = fs.readFileSync('encrypted.crypzor');

// Decrypt the encrypted file
const decipher = crypto.createDecipheriv('aes-128-cbc', key, iv);
let decrypted = decipher.update(encryptedFile);
decrypted = Buffer.concat([decrypted, decipher.final()]);
console.log('Decrypted text: ' + decrypted.toString());

File: decoder.py
import os
import sys
import base64
import hashlib
from Crypto.Cipher import AES

# Load the key and IV from the file
with open("key.keyzor", "rb").read() as key_file:
    key = key_file[:16]
    iv = key_file[16:]

# Load the encrypted file
with open("encrypted.crypzor", "rb").read() as encrypted_file:

    # Decrypt the encrypted file
    cipher = AES.new(key, AES.MODE_CBC, iv)
    decrypted = cipher.decrypt(encrypted_file)

# Remove padding
pad = decrypted[-1]
decrypted = decrypted[:-pad]

print("Decrypted text:", decrypted.decode())
input("Press Enter to continue...")

